docs/Advanced-Usage/Debugging/RTL-Simulation.rst

```diff
@@ -102,7 +102,14 @@ Run all RISCV-tools assembly and benchmark tests on a verilated simulator.
     make DESIGN=FireSimNoNIC
     make DESIGN=FireSimNoNIC -j run-asm-tests
     make DESIGN=FireSimNoNIC -j run-bmark-tests
+    
+Run all RISCV-tools assembly and benchmark tests on a verilated simulator with waveform dumping.
 
+::
+
+    make DESIGN=FireSimNoNIC verilator-debug
```

Comment:
you should be able to just `make DESIGN=FireSimNoNIC`, and the the latter recipes will make sure the simulator is built (verilator by default).